lib/views/BookmarkListView.dart

```diff
@@ -92,11 +133,15 @@ class BookmarkListView
             },
             child: Row(
               children: <Widget>[
+                Padding(
+                    padding: EdgeInsets.only(left: 10),
+                    child: bookmark.favicon),
                 Expanded(
                   child: ListTile(
-                    title: Text("${bookmark.title}"),
+                    title: Text(
+                        "${this._capString(bookmark.title, BookmarkListView.titleLength)}"),
                     subtitle: Text(
-                        "${bookmark.description != "" ? bookmark.description : bookmark.url}"),
+                        "${this._capString(bookmark.description != "" ? bookmark.description : bookmark.url, BookmarkListView.subtitleLength)}"),
```

Comment:
I think it makes sense to refactor this into something like `this._prepareSubtitle(bookmark)`. WDYT?

Reply:
Ah, good idea! :)